_data/publications.yml

```diff
@@ -14330,3 +14330,60 @@
   type: article
   uri: http://publications.imp.fu-berlin.de/id/eprint/962
   userid: 167
+---
```

Comment:
this won't work because it's automatically updated; I think the status was that we will add the stuff that's missing to the FU publications list (dissertations are also missing).

Until then, there is just no citation for vaquita

Reply:
Or just add a link to the free text in the meantime

Reply:
So I remove everything from the yml again?

Reply:
Yes, because it will be removed anyway when the yml is auto-updated

Reply:
How do  I add a link to a free text?

Reply:
Just markdown in the tool description